fulp_modules/bloodsuckers/code/antagonists/bloodsucker_datum.dm

```diff
@@ -7,6 +7,7 @@
 	show_name_in_check_antagonists = TRUE
 	can_coexist_with_others = FALSE
 	hijack_speed = 0.5
+	// tips = BLOODSUCKER_TIPS
```

Comment:
There's some coding changes I want to do before bloodsuckers / monster hunters get added, please leave them uncommented and I'll do them once this PR is merged